modules/nw-ingress-sharding.adoc

```diff
@@ -6,25 +6,35 @@
 [id="nw-ingress-sharding_{context}"]
 = Ingress Controller sharding
 
-You can use Ingress sharding, also known as router sharding, to distribute a set of routes across multiple routers by adding labels to routes, namespaces, or both. The Ingress Controller uses a corresponding set of selectors to admit only the routes that have a specified label. Each Ingress shard comprises the routes that are filtered using a given selection expression.
+You can use Ingress sharding, also known as router sharding, to distribute a set of routes across multiple routers by adding labels to routes, namespaces, or both. The Ingress Controller uses a corresponding set of selectors to admit only the routes that have a specified label. Each Ingress shard comprises the routes that are filtered by using a given selection expression.
 
 As the primary mechanism for traffic to enter the cluster, the demands on the Ingress Controller can be significant. As a cluster administrator, you can shard the routes to:
 
-* Balance Ingress Controllers, or routers, with several routes to speed up responses to changes.
-* Allocate certain routes to have different reliability guarantees than other routes.
+* Balance Ingress Controllers, or routers, with several routes to accelerate responses to changes.
+* Assign certain routes to have different reliability guarantees than other routes.
 * Allow certain Ingress Controllers to have different policies defined.
 * Allow only specific routes to use additional features.
 * Expose different routes on different addresses so that internal and external users can see different routes, for example.
-* Transfer traffic from one version of an application to another during a blue green deployment.
+* Transfer traffic from one version of an application to another during a blue-green deployment.
 
-When Ingress Controllers are sharded, a given route is admitted to zero or more Ingress Controllers in the group. A route's status describes whether an Ingress Controller has admitted it or not. An Ingress Controller will only admit a route if it is unique to its shard.
+When Ingress Controllers are sharded, a given route is admitted to zero or more Ingress Controllers in the group. The status of a route describes if an Ingress Controller has admitted the route or not. An Ingress Controller only admits a route if the route is unique to a shard.
```

Comment:
I think you need "whether" in this sentence. If you move it around, you would get:

"whether or not an Ingress Controller has admitted the route"

whereas
"if or not an Ingress Controller has admitted the route"
is ungrammatical.

```suggestion
When Ingress Controllers are sharded, a given route is admitted to zero or more Ingress Controllers in the group. The status of a route describes whether an Ingress Controller has admitted the route or not. An Ingress Controller only admits a route if the route is unique to a shard.
```

Reply:
You could actually omit the "or not" for minimalism - _whether_ implies _or not_:
```suggestion
When Ingress Controllers are sharded, a given route is admitted to zero or more Ingress Controllers in the group. The status of a route describes whether an Ingress Controller has admitted the route. An Ingress Controller only admits a route if the route is unique to a shard.
```

Reply:
Thanks, "whether" would also be ungrammatical according to the IBM SG:

![Screenshot from 2025-02-11 15-44-45](https://github.com/user-attachments/assets/3f427ba4-5ceb-42ee-8487-b669edbc6f0f)

Reply:
Possibly:

> The status of a route describes if an Ingress Controller admits the route or did not admit the route

Reply:
ah yeah - it says not to use "whether or not" and instead to use whether. So this it should be my [second suggestion](https://github.com/openshift/openshift-docs/pull/87848#discussion_r1951094098): "The status of a route describes whether an Ingress Controller has admitted the route."